iree/compiler/Dialect/HAL/Target/LLVM/LLVMTargetOptions.cpp

```diff
@@ -82,6 +82,13 @@ LLVMTargetOptions getLLVMTargetOptionsFromFlags() {
     llvmTargetOptions.targetCPUFeatures = clTargetCPUFeatures;
   }
 
+  static llvm::cl::opt<std::string> clSanitizerKind(
+      "iree-llvm-sanitize",
+      llvm::cl::desc("Specify LLVM sanitize feature such as 'address' ETC"),
+      llvm::cl::init(""));
+  if (clSanitizerKind == "address")
+    llvmTargetOptions.sanitizerKind = iree_Sanitizer_kAddress;
+
```

Comment:
Let's add an else with an error if the sanitize kind is not recognized (I just hit this by guessing the value of "asan" instead of "address").

Reply:
Thanks, then what can I use to show the error message? Can I just use printf..?

Reply:
`llvm::errs () <<` in a pinch. It is also possible to set up the command line option machinery to know about the values. There are some examples somewhere (afk right now), but the docs are not too bad: https://llvm.org/docs/CommandLine.html#id12

If you use the built-in support for alternates, it will print the error messages for you, iirc.

Reply:
I prefer setting up the command line option. This is more clear to me.

https://llvm.org/docs/CommandLine.html#selecting-an-alternative-from-a-set-of-possibilities

Here is another example in IREE:

https://github.com/google/iree/blob/main/iree/compiler/Dialect/VM/Conversion/TargetOptions.cpp#L34-L41

note that the IREE example is using a list, and you don't need a list for this case.